source/testing-your-html/index.html.md.erb

```diff
@@ -45,7 +47,10 @@ Each object in the `fixtures` list is a different example of the component, wher
 - `name` is the name of the example
 - `options` are the options that generate this example's `html`
 - `html` is the HTML that GOV.UK Frontend outputs with these options
+- `hidden` is `true` if the fixture does not look or behave differently to the other fixtures provided
```

Comment:
who is providing the fixtures?

Reply:
We are – the first sentence in the document starts 'You can use our test fixtures to check you're outputting the same HTML that GOV.UK Frontend uses'